modules/apps/29-fee/ibc_module.go

```diff
@@ -200,7 +200,7 @@ func (im IBCModule) OnRecvPacket(
 
 	// incase of async aknowledgement (ack == nil) store the ForwardRelayer address for use later
 	if ack == nil && found {
-		im.keeper.SetForwardRelayerAddress(ctx, channeltypes.NewPacketId(packet.GetSourceChannel(), packet.GetSourcePort(), packet.GetSequence()), forwardRelayer)
+		im.keeper.SetForwardRelayerAddress(ctx, channeltypes.NewPacketId(packet.GetSourceChannel(), packet.GetSourcePort(), packet.GetSequence()), relayer.String())
```

Comment:
We're technically not setting the ForwardRelayerAddress anymore, I'm wondering if we should have a different setter for this use case specifically.

Reply:
Yes maybe we should name it something else?

Reply:
Is this mapping only used for async acks? Maybe the function name should take that into account? `SetRelayerAddressForAsyncAck`

Reply:
I went with `SetRelayerAddressForAsyncAck` & respective getter fn